rr_control_input_manager/scripts/ps4_mapper.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
```

Comment:
As a future note I think it is best practice to try and avoid pulling in two features on one PR.
I would recommend for best practice opening a feature branch and merging this in separately in the future.